experimental/dds/tree2/src/test/snapshots/files/empty-root-final.json

```diff
@@ -14,7 +14,101 @@
               "String": {
                 "type": "blob",
                 "content": {
-                  "trunk": [],
+                  "trunk": [
+                    {
+                      "change": {
+                        "changes": [
+                          {
+                            "schema": {
+                              "new": {
+                                "version": "1.0.0",
```

Comment:
the lack of versions at lower levels of indent in this file is a bit concerning.
What version do you bump when making "old" not included for example? I don't think that needs to block this PR, but is something we really need to sort out.

Reply:
Will file an item before merging.

Reply:
Item filed.